### PR TITLE
chore: prepare for 0.15.0 release

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
It bumps the version of the library in the docs site, and adds named parameters in a function of the public API.

## Why is it important?
Keep docs up-to-date, and increase the readability of that function while consuming it.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -60,4 +60,4 @@ nav:
     - Getting help: getting_help.md
 edit_uri: edit/main/docs/
 extra:
-    latest_version: 0.14.0
+    latest_version: 0.15.0
```
Comment (mdelapenya) [inline mkdocs.yml:63]: I've double checked that this value is not affecting the web UI

Screenshot 2022-10-24 at 11 34 42

File: wait/sql.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ import (
 const defaultForSqlQuery = "SELECT 1"
 
 //ForSQL constructs a new waitForSql strategy for the given driver
-func ForSQL(port nat.Port, driver string, url func(string, nat.Port) string) *waitForSql {
+func ForSQL(port nat.Port, driver string, url func(host string, port nat.Port) string) *waitForSql {
```
Comment (mdelapenya) [inline wait/sql.go:15]: I'm adding this cosmetic change here, but I can send it in a separate PR if needed

**Comment by mdelapenya** [comment]
Merging as the change is harmless

File: wait/sql.go (hunk 2 above)
Comment (kat1906) [reply wait/sql.go:15]: this is a breaking change?

**Pull request merged**